phpcs-sniffs/Formidable/Sniffs/CodeAnalysis/RedundantEmptyOnAssignedVariableSniff.php

```
@@ -556,6 +653,77 @@ private function isInsideConditionalBlock( File $phpcsFile, $functionToken, $tok
 
+	/**
+	 * Check if two tokens are in the same conditional scope.
+	 *
+	 * @param File $phpcsFile The file being scanned.
+	 * @param int  $pos1      First token position.
+	 * @param int  $pos2      Second token position.
+	 *
+	 * @return bool True if in same conditional scope, false otherwise.
+	 */
+	private function areInSameConditionalScope( File $phpcsFile, $pos1, $pos2 ) {
+		$tokens = $phpcsFile->getTokens();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$tokens'. (undefined)

(UnusedLocalVariable)